pkg/kubelet/kuberuntime/kuberuntime_gc.go

```diff
@@ -210,11 +216,6 @@ func (cgc *containerGC) evictableContainers(minAge time.Duration) (containersByE
 		evictUnits[key] = append(evictUnits[key], containerInfo)
 	}
 
-	// Sort the containers by age.
```

Comment:
Recording some notes because these types of PRs are hard to review without looking at a lot of detail.

`evictableContainers` is invoked by `evictContainers` and iterates on the list.

Reply:
it later takes that list and does a `removeOldestN` in `enforceMaxContainersPerEvictUnit` which in turn takes evictUnits and puts it into a sort flattened list before calling `removingOldestN`.

Reply:
Thanks @derekwaynecarr 
Should have put the call chain the description so that review time can be saved.